### PR TITLE
Update wallet address in URL after changing account

### PR DESCRIPTION
**Pull request by michalsmiarowski**
When changing account in MetaMask we also have to change the wallet address in
the URL so after refresh the dApp will log into the same account.

### CODE REVIEW

File: solidity-v1/dashboard/src/hooks/useAutoWalletAddressInjectIntoUrl.js [hunk 1]
```diff
@@ -13,12 +14,27 @@ const useAutoWalletAddressInjectIntoUrl = () => {
   const locationHasChanged = useHasChanged(location.pathname)
 
   useEffect(() => {
+    console.log("USE EFFECTB ROS")
```
[the remark this answers is not in the document]
Comment (michalsmiarowski) [reply solidity-v1/dashboard/src/hooks/useAutoWalletAddressInjectIntoUrl.js]: ef265806bf22007c82bb48e69ccedbacca192581

File: solidity-v1/dashboard/src/hooks/useAutoWalletAddressInjectIntoUrl.js [hunk 2]
```diff
@@ -13,12 +14,27 @@ const useAutoWalletAddressInjectIntoUrl = () => {
...
     if (locationHasChanged) return
+    console.log("location.pathname", location.pathname)
```
[the remark this answers is not in the document]
Comment (michalsmiarowski) [reply solidity-v1/dashboard/src/hooks/useAutoWalletAddressInjectIntoUrl.js]: ef265806bf22007c82bb48e69ccedbacca192581

File: solidity-v1/dashboard/src/hooks/useAutoWalletAddressInjectIntoUrl.js [hunk 3]
```diff
@@ -13,12 +14,27 @@ const useAutoWalletAddressInjectIntoUrl = () => {
...
     // change url to the one with address when we connect to the explorer mode
     if (!walletAddressFromUrl && connector && yourAddress) {
       const newPathname = "/" + yourAddress + location.pathname
       history.push({ pathname: newPathname })
     }
+
+    if (!walletAddressFromUrl && connector && yourAddress) {
+      const newPathname = "/" + yourAddress + location.pathname
+      history.push({ pathname: newPathname })
```
[the remark this answers is not in the document]
Comment (michalsmiarowski) [reply solidity-v1/dashboard/src/hooks/useAutoWalletAddressInjectIntoUrl.js]: 9bedb56260285c8700916a4f91862e3bcd23ad6d

**Review by r-czajkowski — APPROVED** [review]
👍

**Pull request merged**